.about.yml

```diff
@@ -0,0 +1,116 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: dolores-landingham-bot
+
+# Full proper name of the project (required)
+full_name: Dolores Landingham Slack Bot
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: project
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+parent: wg-onboarding
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live, sunset, transfer, end
+stage: live
+
+# Description of the project
+description: This is a Slack bot that helps onboard new hires at 18F.
+
+# Tags that describe the project or aspects of the project
+tags:
+- onboarding
+- slack
+- bot
+- dolores
+- landingham
+
+# Should this repo have automated tests? If so, set to `true`. (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: jessieay
+  role: lead
+- github: gemfarmer
+- github: ccostino
+- github: monfresh
+- github: melodykramer
+- github: brittag
+- github: danielmorrison
+- github: adelevie
+
+# Organizations or individuals who have adopted the project for their own use
+# Items:
+# - id: The name of the organization or individual
+#   url: A URL to the user's version of the project
+#users:
+#-
+
+# Brief descriptions of significant project developments
+#milestones:
+#-
+
+# Technologies used to build the project
+stack:
+- Ruby
+- Rails
+- Clockwork
+- Timecop
+- RSpec
+- Slack API
```

Comment:
Hmm, not sure why the update isn't showing up here, but MyUSA Authentication is a part of this list now (viewing the file itself shows it).